test-basic/plan-processors.js

```diff
@@ -314,4 +317,14 @@ describe('processor', function() {
       })
     .catch(done);
   });
+  it('generateView with empty schema', function(done) {
+      assert.throws(function () { generateView(
+          p.fromLiterals([
+              {id:1, val: 2},
+              {id:2, val: 4}
+          ]),
+          '', 'xml', ''
+      );}, Error, 'schema cannot be null or empty');
+      done();
+  });
 });
```

Comment:
It will be good to add a few unit tests for the redaction functions as well, but that will need to wait until I generate the bind() function for the Node.js API.  Just for background, here are the unit tests on the enode:

http://svn.marklogic.com/trac.engsvn/browser/xdmp/trunk/src/jstest/optictest/test-basic/redact.js#L207

Similarly, after I add the enode documentation for the redaction functions, it will be good to add links from the JSDoc for the Node.js client redaction functions.